test/functional/testlib.bash

```diff
@@ -4,10 +4,18 @@ FUNC_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 TEST_FILENAME=$(basename "$BATS_TEST_FILENAME")
 TEST_NAME=${TEST_FILENAME%.bats}
 THEME_DIRNAME="$BATS_TEST_DIRNAME"
+CACERT_DIR="$FUNC_DIR/test_certificates" # trusted key store path
+CLIENT_CERT_DIR="$TEST_NAME/target-dir/etc/swupd"
```

Comment:
Since this environment variable depends on the value of $TEST_NAME to already exist, I think this variable should be defined within the set_env_variables https://github.com/clearlinux/swupd-client/blob/master/test/functional/testlib.bash#L148, that is its purpose.